web/client/components/mapcontrols/scale/ScaleBox.jsx

```diff
@@ -37,7 +37,7 @@ var ScaleBox = React.createClass({
     },
     onComboChange(event) {
         var selectedZoomLvl = parseInt(event.nativeEvent.target.value, 10);
-        this.props.onChange(selectedZoomLvl, this.props.scales[selectedZoomLvl]);
```

Comment:
This is meant to be used by "alternative" action bound to onChange. Does the additional parameter create any issue for you?